src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -187,9 +187,16 @@ internal void DependencyRegen(WorkspaceModel ws, bool forceCompute = false)
                 foreach (var package in dependencyViewExtension.pmExtension.PackageLoader.LocalPackages.Where(x => 
                 x.LoadState.ScheduledState == PackageLoadState.ScheduledTypes.ScheduledForDeletion || x.LoadState.ScheduledState == PackageLoadState.ScheduledTypes.ScheduledForUnload))
                 {
-                    (packageDependencies.FirstOrDefault(x => x.Name == package.Name) as PackageDependencyInfo).State =
+                    try
+                    {
+                        (packageDependencies.FirstOrDefault(x => x.Name == package.Name) as PackageDependencyInfo).State =
                         PackageDependencyState.RequiresRestart;
-                    hasPackageMarkedForUninstall = true;
+                        hasPackageMarkedForUninstall = true;
+                    }
+                    catch (Exception ex)
+                    {
+                        dependencyViewExtension.OnMessageLogged(LogMessage.Info(string.Format(Properties.Resources.DependencyViewExtensionErrorTemplate, ex.ToString())));
```

Comment:
perhaps it would be good to note in the message what failed to happen - so If I am correct about this code, it would be something like "failure to set package uninstall state" ? - I think it's acceptable for the text to be non localized if the message is not intended for users but more for debugging for us.

Reply:
Done.